lib/cc/cli/prepare/quality.rb

```diff
@@ -0,0 +1,59 @@
+require "safe_yaml/load"
+
+module CC
+  module CLI
+    class Prepare
+      class Quality < Command
```

Comment:
This'll only execute if `codeclimate prepare:quality` is run. We'll run it (guarded by rollout) on hosted.